purchase_request_department/tests/test_purchase_request_department.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017-2020 Forgeflow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPurchaseRequest(TransactionCase):
+    def setUp(self):
```

Comment:
LGTM. I agree with @flalexg comment, modify setup to setupclass and self to cls.